src/dashboard/apps/home/apps.py

```diff
@@ -4,7 +4,8 @@
 
 
 class HomeConfig(AppConfig):
-    """Home app config."""
+    """home app config."""
```

Comment:
Title case is better for a docstring right?

Reply:
✅